components/brave_news/browser/feed_v2_builder.h

```diff
@@ -41,14 +46,25 @@ class FeedV2Builder {
   FeedV2Builder& operator=(const FeedV2Builder&) = delete;
   ~FeedV2Builder();
 
-  void Build(bool recalculate_signals, BuildFeedCallback callback);
+  void AddListener(mojo::PendingRemote<mojom::FeedListener> listener);
+
   void BuildChannelFeed(const std::string& channel, BuildFeedCallback callback);
   void BuildPublisherFeed(const std::string& publisher_id,
                           BuildFeedCallback callback);
+  void BuildAllFeed(BuildFeedCallback callback);
 
   void GetSignals(GetSignalsCallback callback);
 
  private:
+  using UpdateCallback = base::OnceCallback<void()>;
+  struct UpdateSettings {
```

Comment:
Can we have a comment like,

```
When a member is set true, the data would be refetched.
```

Or changing member name to more explicit name?

```
bool recalc_signals;
bool refetch_topics;
```

Reply:
Added a comment!